### PR TITLE
Nh rgb to hex conversion

### PR DESCRIPTION
**Pull request by nxnode**

### CODE REVIEW

**Review by W-Ely — DISMISSED** [review]
🍠

**Comment by nxnode** [comment]
When I updated the readme to fix conflict, it apparently dismissed your review and now requires another approval. Which I guess makes sense, since I did update another file. Sorry! Should've did it earlier!

**Review by W-Ely — APPROVED** [review]

**Pull request merged**
